docs/resources.md

```diff
@@ -618,6 +619,213 @@ the container image
+            ]
+          }
+        ]
+      },
+      "serviceAccount": "default",
+      "taskRef": {
+        "name": "source-to-image",
+        "kind": "Task"
+      },
+      "timeout": "1h0m0s"
+    },
+    "status": {
+      "conditions": [
+        {
+          "type": "Succeeded",
+          "status": "True",
+          "lastTransitionTime": "2019-07-04T11:03:52Z"
+        }
+      ],
+      "podName": "pipeline-run-api-16aa55-source-to-image-task-rpndl-pod-091933",
+      "startTime": "2019-07-04T10:56:40Z",
+      "completionTime": "2019-07-04T11:03:52Z",
+      "steps": [
+        {
+          "terminated": {
+            "exitCode": 0,
+            "reason": "Completed",
+            "startedAt": "2019-07-04T10:56:45Z",
+            "finishedAt": "2019-07-04T11:03:51Z",
+            "containerID": "containerd://5cd26e4f1a4aa01c91cb4d4f8f7a2533e41ecf70ffa0618e6773a1f8dd9bb591"
+          },
+          "name": "build-and-push"
+        },
+        {
+          "terminated": {
+            "exitCode": 0,
+            "reason": "Completed",
+            "startedAt": "2019-07-04T10:56:46Z",
+            "finishedAt": "2019-07-04T11:03:51Z",
+            "containerID": "containerd://002908c1d71410e4e214c3b529705b14558dd53b1d748f5beda7e489da84423f"
+          },
+          "name": "build-index-json"
+        },
+        {
+          "terminated": {
+            "exitCode": 0,
+            "reason": "Completed",
+            "startedAt": "2019-07-04T10:56:43Z",
+            "finishedAt": "2019-07-04T10:58:17Z",
+            "containerID": "containerd://b87f0f5a33944c35be7bbe9960b02a0ad9bd94b2a98cf542fd7077bb0a5ad9bc"
+          },
+          "name": "git-source-git-resource-16aa55-xcfwz"
+        },
+        {
+          "terminated": {
+            "exitCode": 0,
+            "reason": "Completed",
+            "startedAt": "2019-07-04T10:56:45Z",
+            "finishedAt": "2019-07-04T11:03:51Z",
+            "containerID": "containerd://fdee40adedfdbbc9c287cebccb5f1f00a5de6ad21a460ca06723822e29a15ccd"
+          },
+          "name": "image-digest-exporter-build-and-push-p5cnn"
+        },
+        {
+          "terminated": {
+            "exitCode": 0,
+            "reason": "Completed",
+            "startedAt": "2019-07-04T10:56:46Z",
+            "finishedAt": "2019-07-04T11:03:51Z",
+            "containerID": "containerd://e74fc527d00f35896ed5033dd3060988c938f6c4d9debe211fa698c5bc3d1c24"
+          },
+          "name": "image-digest-exporter-build-index-json-bcthp"
+        },
+      ],
+      "cloudEvents": [
+        {
+          "target": "http://sink:8080/",
+          "status": {
+            "condition": "Unknown",
+            "message": "",
+            "retryCount": 0
+          }
+        },
+      ],
+      "resourcesResult": [
+        {
+          "name": "image-resource-api-16aa55",
+          "digest": "sha256:e4fc840de12da28174a5c22da0b65ecb3b9bd5ec3f3fa83585e04e7a5509eb7c"
+        }
+      ]
+    }
+  }
+```
```

Comment:
i feel like this is a bit long to embed directly in the markdown like this 🤔  what do you think about including on a subset of it, and/or maybe including the complete example in a separate file linked from here?

Reply:
good point, I'm not sure we need a full TaskRun example - as that may become stale anyways - headers and basic data structure should be enough.